lib/inventoryware/utils.rb

```diff
@@ -59,12 +59,7 @@ def self.check_file_location?(path)
 
     # raise an error if given path isn't a directory
     def self.exit_unless_dir(path)
-      unless File.directory?(path)
-        raise FileSysError, <<-ERROR.chomp
-Directory #{File.expand_path(path)} not found.
-Please create it before continuing"
-        ERROR
-      end
+      Dir.mkdir_p(path) unless File.directory?(path)
```

Comment:
We probably ought to raise an exception if `path` exists but is not a directory.

Reply:
ditto